packages/opentelemetry-node-sdk/src/NodeTracer.ts

```diff
@@ -28,15 +28,18 @@ export class NodeTracer extends BasicTracer {
   /**
    * Constructs a new Tracer instance.
    */
-  constructor(config: NodeTracerConfig) {
-    if (config.scopeManager === undefined) {
-      config.scopeManager = new AsyncHooksScopeManager();
-      config.scopeManager.enable();
+  constructor(config?: NodeTracerConfig) {
```

Comment:
What about `config: NodeTracerConfig = {}`?

Reply:
Could you also update the `NodeTracer` tests?  Use `tracer = new NodeTracer();` instead of `tracer = new NodeTracer({});`?

Reply:
> What about `config: NodeTracerConfig = {}`?

Yes and it's more consistent with `BasicTracer`

Reply:
> > What about `config: NodeTracerConfig = {}`?
> 
> Yes and it's more consistent with `BasicTracer`

@mayurkale22 

if I do 
```
tracer = new NodeTracer(null); // this will throw
```
Should I let the check  ?
```
const internalConfig = config || {};
```

Reply:
Ok makes sense 💯

Reply:
I think we are okay with `new NodeTracer(null)` failing because the typings should complain anyways. If we force it to allow `null`, thn I think BasicTracer should allow it as well. WDYT?

Reply:
Yep. I let the same behaviour than BasicTracer